### PR TITLE
Add ru locale.

### PR DESCRIPTION
**Pull request by alek13**

### CODE REVIEW

File: locale/areas.ru.tr [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+<ID> [faction_name]=<ID> [имя_фракции]
+<NewOwner>=<новый_владелец>
+<ParentID>=<ID_родительской>
+<PlayerName>=<имя_игрока>
+<newName>=<новое_имя>
+@1 has given you control over the area "@2" (ID @3).=@1 передал вам территорию "@2" (ID @3).
+@1 spanning up to @2x@3x@4.=@1 размером до @2x@3x@4.
+A regular expression is required.=Не указано регулярное выражение.
+Area @1 does not exist or is not owned by you.=Территория @1 не существует или Вы не имеете к ней доступа.
+Area closed for faction members.=Территория закрыта для фракций.
```
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation `Area closed for factions.` add `members` if possible

File: locale/areas.ru.tr [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Faction doesn't exists=Фракция не существует.
+Find areas using a Lua regular expression=Поиск территорий с использованием регулярных выражений Lua
+Get information about area configuration and usage.=Получить информацию о конфигурации и использовании защиты территорий.
+
+Give a player access to a sub-area between two positions that have already been protected, Use set_owner if you don't want the parent to be set.=Дать игроку доступ к выделенной территории, создав дополнительный регион. Используйте set_owner, если вы не хотите привязки к родительской территории.
+
+Invalid regular expression.=Неверное регулярное выражение.
+Limit: @1 areas=Ограничение: @1 территорий(я)
+Limit: no area count limit=Ограничение: нет ограничений на кол-во территорий
+List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий (или всех существующих территорий, если Вы имеете привилегию areas)
```
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation  `List your available territories (or all existing territories if you have the areas privilege).` may try to match the `or all areas if you are an admin` part better

File: locale/areas.ru.tr [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+Move (or resize) an area to the current positions.=Переместить (или изменить размер) территории.
+No matches found.=Совпадений не найдено.
+No visible areas.=Нет доступных территорий.
+Owner changed.=Владелец изменён.
+Players with the "@1" privilege can protect up to @2 areas=Игроки, имеющие привилегию "@1" могут защищать до @2 территорий
+
+Protect an area between two positions and give a player access to it without setting the parent of the area to any existing area=Защитить территорию между двумя позициями и дать игроку доступ к ней без привязки к родительскому региону.
+
+Protect your own area=Защитить территорию и установить себя владельцем
```
Comment (Niklp09) [inline locale/areas.ru.tr]: `Protect your own area=Protect an area and set yourself as the owner of it` doesn't match very well but seems ok for me

File: locale/areas.ru.tr [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+### legacy.lua ###
+
+<version>=<версия>
+Converted areas saved. Done.=Готово. Сконвертированные территории сохранены.
+Converting areas…=Конвертирование территорий…
+Error loading legacy file: @1=Ошибка загрузки файла с устаревшим форматом: @1
+Invalid version number. (0 allowed)=Неверный номер версии. (поддерживается 0)
+Legacy file loaded.=Файл с устаревшим форматом загружен.
+
+Loads, converts, and saves the areas from a legacy save file.=Загружает, конвертирует и сохраняет файл с устаревшим форматом.
```
Comment (Niklp09) [inline locale/areas.ru.tr]: your translation `Loads, converts, and saves a legacy file.` (`the areas from` is missing)

File: locale/areas.ru.tr [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Table format updated.=Формат обновлён.
+unnamed=<без_названия>
```
Comment (Niklp09) [inline locale/areas.ru.tr]: no `<>` needed

**Review by Niklp09 — COMMENTED** [review]
Left some comments, everything else LGTM. I used deepl for translating your translations back to english. Please note, I don't speak russian - a native speaker should check this too.

**Comment by SmallJoker** [comment]
@alek13 The PR currently waits on your feedback on the suggestions above. Feel free to reject such that are not justified so that we can proceed with this PR towards merge.

**Comment by alek13** [comment]
@SmallJoker / @Niklp09 Sorry, I just forget for it. Will take a look at nearest time

**Comment by ssdaniel24** [comment]
Fixed all remarks above.

File: locale/areas.ru.tr [hunk 6]
```diff
@@ -14,7 +14,7 @@
 @1 spanning up to @2x@3x@4.=@1 размером до @2x@3x@4.
 A regular expression is required.=Не указано регулярное выражение.
 Area @1 does not exist or is not owned by you.=Территория @1 не существует или Вы не имеете к ней доступа.
-Area closed for faction members.=Территория закрыта для фракций.
+Area closed for faction members.=Территория закрыта для игроков из фракций.
```
Comment (alek13) [inline locale/areas.ru.tr:17]: "игроков из" -- прикольно, а то "членов" звучит так себе :)))

File: locale/areas.ru.tr [hunk 7]
```diff
@@ -32,7 +32,7 @@ Give a player access to a sub-area between two positions that have already been
 Invalid regular expression.=Неверное регулярное выражение.
 Limit: @1 areas=Ограничение: @1 территорий(я)
 Limit: no area count limit=Ограничение: нет ограничений на кол-во территорий
-List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий (или всех существующих территорий, если Вы имеете привилегию areas)
+List your areas, or all areas if you are an admin.=Вывести список доступных Вам территорий или всех существующих территорий, если вы администратор.
```
Comment (alek13) [inline locale/areas.ru.tr:35]: here is not truth, as I see in code.
maybe I'm wrong, but

you may not be an administrator, but have the privilege area

Comment (alek13) [reply locale/areas.ru.tr:35]: @ssdaniel24 , can you double check?

**Comment by ssdaniel24** [comment]
*Force push for fixing my git credentials*

File: locale/areas.ru.tr (hunk 7 above)
Comment (ssdaniel24) [reply locale/areas.ru.tr:35]: Maintainers wrote it like this. Who are we to change it? We are translating only.

Comment (alek13) [reply locale/areas.ru.tr:35]: it could be just a mistake. Why not to fix this?

Comment (ssdaniel24) [reply locale/areas.ru.tr:35]: If it is mistake, it is in repo, not in the translation.

**Comment by alek13** [comment]
@SmallJoker / @Niklp09 fixed.

**Comment by Niklp09** [comment]
Thanks, I'll take a look at your changes in the next days.

**Review by Niklp09 — APPROVED** [review]
Checked your new changes w/ deepl, LGTM. Will merge in a few days unless there are objections.
edit: typo

**Comment by alek13** [comment]
:+1:  Danke schön :)

**Pull request merged**